src/resources/views/video/edit.blade.php

```diff
@@ -0,0 +1,28 @@
+@extends('appshell::layouts.private')
```

Comment:
Why is this not a modal like the create video? Please make this a modal as well.